x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_initializer.tsx

```diff
@@ -82,24 +118,36 @@ export const AnomalySwimlaneInitializer: FC<AnomalySwimlaneInitializerProps> = (
       (swimlaneType === SWIMLANE_TYPE.VIEW_BY && !!viewBySwimlaneFieldName));
 
   const resultInput = {
+    jobIds,
     panelTitle,
     swimlaneType,
     ...(viewBySwimlaneFieldName ? { viewBy: viewBySwimlaneFieldName } : {}),
   };
 
   return (
-    <EuiModal initialFocus="[name=panelTitle]" onClose={onCancel}>
-      <EuiModalHeader>
-        <EuiModalHeaderTitle>
-          <FormattedMessage
-            id="xpack.ml.swimlaneEmbeddable.setupModal.title"
-            defaultMessage="Anomaly swim lane configuration"
-          />
-        </EuiModalHeaderTitle>
-      </EuiModalHeader>
+    <>
+      <EuiFlyoutHeader>
+        <EuiTitle>
+          <h2>
+            <FormattedMessage
+              id="xpack.ml.swimlaneEmbeddable.setupModal.title"
+              defaultMessage="Anomaly swim lane configuration"
+            />
+          </h2>
+        </EuiTitle>
+      </EuiFlyoutHeader>
 
-      <EuiModalBody>
+      <EuiFlyoutBody>
         <EuiForm>
+          <JobSelectorControl
```

Comment:
Should we allow the user to apply the changes if no job is selected? Currently this acts to show all jobs, but I prefer the way it works inside the alerting rule flyout where it forces you to make a selection?

<img width="676" alt="Screenshot 2024-05-07 at 12 47 44" src="https://github.com/elastic/kibana/assets/7405507/04444d48-3760-4c59-9b48-52671d20cee0">

Do we need a quick way to select all jobs / groups?

Reply:
Updated in https://github.com/elastic/kibana/pull/182586/commits/1c6cc4ef4d6f9a522e6d7e18b3d952ef0adc533e.
Added a validation for checking that at least one job is selected.